### PR TITLE
DEV-1801: Changes around verbose and updates in config

### PR DESCRIPTION
**Pull request by m-terel**

### CODE REVIEW

File: rportd.example.conf [hunk 1]
```diff
@@ -210,7 +210,8 @@
   ## If specified, rportd will serve static files from this directory on the same API address.
   ## This is the place where the frontend files (html/js) go.
   ## Learn how to install the frontend https://github.com/cloudradar-monitoring/rport/blob/master/docs/frontend.md
-  doc_root = "/var/lib/rport/docroot"
+  #doc_root = "C:\Program Files\rport\docroot"
```
Comment (matematik7) [inline rportd.example.conf:213]: Other entries have space after `#`.

File: share/logger.go [hunk 2]
```diff
@@ -15,6 +16,9 @@ const (
 )
 
 func ParseLogLevel(str string) (LogLevel, error) {
+	if str == "" {
+		return LogLevelError, errors.New("log level not specified")
```
Comment (matematik7) [inline share/logger.go:20]: Not sure why this is needed since we have a default log level?

**Review by matematik7 — APPROVED** [review]

File: rportd.example.conf (hunk 1 above)
Comment (m-terel) [reply rportd.example.conf:213]: Actually, not all of them. It's mixed, some with space, some without.

Comment (matematik7) [reply rportd.example.conf:213]: Well, I don't really care one way or another. Do you mind making it consistent?

File: share/logger.go (hunk 2 above)
Comment (m-terel) [reply share/logger.go]: Removed.

File: rportd.example.conf (hunk 1 above)
Comment (m-terel) [reply rportd.example.conf:213]: done

**Review by matematik7 — APPROVED** [review]

**Pull request merged**